### ISSUE TITLE
Feature: make location sensor a device tracker

### ISSUE THREAD

**Issue by Secretowo** [LAB02-Research/HASS.Agent#238]
Having the location sensor be a device tracker instead of just lat and long as a string would enable it showing on the map and allow it to be easily used for geofencing.

**Comment by LAB02-Admin** [comment]
Agree, it's on my todo list: https://lab02research.youtrack.cloud/issue/hassagent-59
